File: Tokopedia/Classses/Object/Shops/Notes/NotesList.m
//
//  NotesList.m
//  Tokopedia
//
//  Created by IT Tkpd on 10/10/14.
//  Copyright (c) 2014 TOKOPEDIA. All rights reserved.
//

#import "NotesList.h"

@implementation NotesList

+(RKObjectMapping *)mapping{
    RKObjectMapping *listMapping = [RKObjectMapping mappingForClass:[NotesList class]];
    [listMapping addAttributeMappingsFromArray:@[@"note_id",
                                                 @"note_status",
                                                 @"note_title"
                                                 ]];
    return listMapping;
}

- (NSString *)note_title
{
    return [_note_title kv_decodeHTMLCharacterEntities];
}

@end

File: Tokopedia/Classses/Object/Shops/Notes/NotesList.swift
//
//  NotesList.swift
//  Tokopedia
//
//  Created by Tedo Pranowo on 7/1/16.
//  Copyright © 2016 TOKOPEDIA. All rights reserved.
//

import Foundation
import RestKit

class NotesList : NSObject{
    
    var note_id : NSString = ""
    var note_status : NSString = ""
    var note_title : NSString = ""
    
    class fileprivate func attributeMappingDictionary() -> [AnyHashable: Any]! {
        return ["note_id" :"note_id",
                "note_status" : "note_status",
                "note_title" : "note_title"]
    }
    
    class func mapping() -> RKObjectMapping! {
        let mapping : RKObjectMapping = RKObjectMapping(for: self)
        
        mapping.addAttributeMappings(from:self.attributeMappingDictionary())
        
        return mapping
    }
    
}
